### PR TITLE
btcli burned_register

### PR DESCRIPTION
**Pull request by camfairchild**
This PR adds the `btcli burned_register` command and a test

Also:
- fixes default netuid to 1 
- fixes inspect when not registered
- switches to subtensor.get_balance for inspect

### CODE REVIEW

File: bittensor/_subtensor/extrinsics/registration.py [hunk 1]
```diff
@@ -178,4 +178,97 @@ def register_extrinsic (
...
+            # create extrinsic call
+            call = substrate.compose_call( 
+                call_module='SubtensorModule',  
+                call_function='burned_register', 
+                call_params={ 
+                    'netuid': netuid,
+                    'hotkey': wallet.hotkey.ss58_address
+                } 
+            )
+            extrinsic = substrate.create_signed_extrinsic( call = call, keypair = wallet.hotkey )
```
Comment (isabella618033) [inline bittensor/_subtensor/extrinsics/registration.py]: should we use coldkey to sign instead? cause burn reg burns tao (stake)

Comment (camfairchild) [reply bittensor/_subtensor/extrinsics/registration.py]: Oops. fixed

File: bittensor/_subtensor/subtensor_impl.py [hunk 2]
```diff
@@ -364,6 +382,11 @@ def kappa (self, netuid: int, block: Optional[int] = None ) -> Optional[int]:
     def difficulty (self, netuid: int, block: Optional[int] = None ) -> Optional[int]:
         if not self.subnet_exists( netuid ): return None
         return self.query_subtensor( "Difficulty", block, [netuid] ).value
+    
+    """ Returns network Burn hyper parameter """
+    def burn (self, netuid: int, block: Optional[int] = None ) -> Optional[bittensor.Balance]:
```
Comment (Eugene-hu) [inline bittensor/_subtensor/subtensor_impl.py:387]: Can we change this naming with something more specific? We might have more parameters later that also require burning

**Review by Eugene-hu — APPROVED** [review]
LGTM, left you a comment

**Pull request merged**